etc/html/HELP.html

```diff
@@ -176,9 +176,9 @@ <h4><a id="class_mem_access"></a>Access (public / protected / private)</h4>
 
 <h4><a id="class_mem_inh"></a>Inheritance</h4>
 You can often access members of a class's base classes, just as if they are defined
-in the derived class. A <a href="http://root.cern.ch/root/html/TH1.html">histogram</a>,
+in the derived class. A <a href="http://root.cern/root/html/TH1.html">histogram</a>,
```

Comment:
Correct link : https://root.cern/doc/master/classTH1.html

Reply:
The other one also works